index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="./styles/defaultStyle.css">
+	<link rel="stylesheet" href="./styles/generalStyle.css">
+	<link rel="stylesheet" href="./styles/style.css">
+	<title>DiegoPortfolio</title>
+</head>
+
+<body>
+	<div class="firstSection">
+		<nav>
+			<ul class="toolbar">
+				<li>John Doe</li>
+				<li><img src="./icons/iconMenu.png" alt="iconMenu"></li>
+			</ul>
+		</nav>
```

Comment:
- [ ] Excellent job creating the header 🥇 , but there are a few semantic errors, Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. Kindly, change `<nav>` tag to `<header>` tag. 💯 
- [ ] The hamburger button should have a `nav` tag. The `nav` tag is for semantics reasons. Kindly, add the `<nav>` tag between `<li>` and `<img>` tags. 🐴